tests/bluetooth/host/cmake.txt

```diff
@@ -0,0 +1,15 @@
+#
+# Common include directories and source files for bluetooth/host unit tests
+#
+
+include_directories(
+  ${ZEPHYR_BASE}/subsys/bluetooth
+  ${ZEPHYR_BASE}/tests/bluetooth/host/host_mocks
+)
+
+FILE(GLOB host_mocks
+  ${ZEPHYR_BASE}/tests/bluetooth/host/host_mocks/assert.c
+)
+
+add_definitions(-include kconfig.h)
+add_definitions(-include ztest.h)
```

Comment:
why do we need this ?

Other tests seems to be fine by using `#include <ztest.h>`
https://github.com/zephyrproject-rtos/zephyr/blob/b88c8e13634437723e294b213876672f31a78e09/tests/kernel/smp/src/main.c#L8

Do you really want `ztest.h` included in ALL source files ?

Reply:
I tried to remove it but compilation process didn't go well as some files that aren't direct project sources require it.
I think this file is required as there is no `autoconf.h`